### ISSUE TITLE
locked monolith vault undoable

### ISSUE THREAD

**Issue by ezdoes** [Iskallia/Vault-public#1202]
**Describe the bug**
The monolith objective need to exit through the portal, but you can't do it with the locked modifier.
I'm thinking make it doable by after you find all the monolith you can go back from the portal.
Maybe remove the locked modifier after find all the monolith to make that work.

**Comment by d0zzer2** [comment]
Did you complete that vault? I’m pretty sure that’s what happens, unless it was just talked about and not implemented yet.

**Comment by ezdoes** [comment]
yes, I completed. But it was in alpha4.1, maybe they already fixed that.

> Did you complete that vault? I’m pretty sure that’s what happens, unless it was just talked about and not implemented yet.
